### PR TITLE
Feature Merge: python scripts to native telegraf inputs

### PR DESCRIPTION
**Pull request by gregschrock**
## Description

Feature merge for moving python scripts to native telegraf inputs.

### CODE REVIEW

File: plugins/inputs/t128_lte/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# 128T LTE Input Plugin
+
+The lte input plugin collects data from a 128T.
```
Comment (agrawalkaushik) [inline plugins/inputs/t128_lte/README.md]: This is a public facing doc so we need to boost this up a bit. We can simply lift up the blurb from [here](https://docs.128technology.com/docs/plugin_monitoring_agent#lte-collector)

File: plugins/inputs/t128_peer_path/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# 128T Peer Path Input Plugin
+
+The peer path input plugin collects data from a 128T instance via graphQL.
```
Comment (agrawalkaushik) [inline plugins/inputs/t128_peer_path/README.md]: This is a public doc so we need to boost it up. Can lift the blurb from [here](https://docs.128technology.com/docs/plugin_monitoring_agent#peer-path-state-collector)

> The peer path input can be used for monitoring the up/down status of all the peer paths on the node

File: plugins/inputs/t128_tank/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# 128T TANK Input Plugin
+
+The tank input plugin collects data from a 128T.
```
Comment (agrawalkaushik) [inline plugins/inputs/t128_tank/README.md:3]: @gregschrock can you recommend a better blurb here.

**Review by agrawalkaushik — APPROVED** [review]

File: plugins/inputs/t128_tank/README.md (hunk 3 above)
Comment (gregschrock) [reply plugins/inputs/t128_tank/README.md:3]:
```
The `t128_tank` input plugin reads from a [TANK](https://github.com/phaistos-networks/TANK) queue and creates metrics using one of the supported [input data formats](/docs/DATA_FORMATS_INPUT.md).
```

**Review by BenMatase — APPROVED** [review]

**Pull request merged**